mitodl/ocw-www
# update this repo with the content output from ocw-studio

#### Pre-Flight checklist


#### What are the relevant tickets?
related to https://github.com/mitodl/ocw-hugo-themes/issues/125

#### What's this PR do?
This PR updates this repo with the latest content generated from `ocw-studio`.  Moving forward, we will be using the `ocw-studio` generated content to build `ocw-www` and this repo will be archived.  The purpose of this PR is just to get the content into a state where it matches the output of `ocw-studio` so the current full site build does not break in the process after merging https://github.com/mitodl/ocw-hugo-themes/pull/126

#### How should this be manually tested?
 - Clone `ocw-hugo-themes` on the `cg/www-image-resource` branch
 - Clone `ocw-hugo-projects`
 - In `ocw-hugo-themes`, run `npm run start:site:webpack`
 - In `ocw-www`, run `hugo server --config /path/to/ocw-hugo-projects/ocw-www/config.yaml --themesDir /path/to/ocw-hugo-themes`
 - Visit http://localhost:1313 and make sure the site renders
